test/pkg/integration/integration.go

```diff
@@ -187,6 +203,43 @@ func Instrument(component ComponentType, agentName string, namespace string, kub
 		return nil, closer, err
 	}
 
+	var (
+		res *rpc.Client
+		cl  []func() error
+	)
+	for i := 0; i < 10; i++ {
```

Comment:
Should we use connectFailureMaxTries instead of 10?

Reply:
Great indicate